lib/dialects/mssql/query-generator.js

```diff
@@ -570,7 +570,14 @@ module.exports = (function() {
 
       if (options.limit || options.offset) {
         if (!options.order || (options.include && !subQueryOrder.length)) {
-          fragment += ' ORDER BY ' + this.quoteIdentifier(model.primaryKeyAttribute);
+          if(!isSubQuery && options.order) {
+            fragment += ', ';
```

Comment:
The referenced issues explained the problem, so I didn't think of describing it again. :smile: